src/caselawclient/xquery/get_components_for_document.xqy

```diff
@@ -0,0 +1,12 @@
+xquery version "1.0-ml";
+
+declare namespace akn = "http://docs.oasis-open.org/legaldocml/ns/akn/3.0";
+
+declare variable $parent_uri as xs:string external;
+declare variable $component as xs:string external;
+
+let $matcher := concat($parent_uri, "/", $component ,"/*.xml")
```

Comment:
I wouldn't necessarily rely on this behaviour; at the moment press summaries all have a URI which begins with the URI of the parent document, but this is a particularly brittle bit of design. I'd be more inclined to look for documents with the right `<meta><neutral_citation>` (I think) value.

Reply:
Good call, I've made a tweak to the query which makes it more robust (I think!)